packages/backend/src/config/app.ts

```diff
@@ -21,6 +21,7 @@ export const Config = {
   logLevel: envString('LOG_LEVEL', 'info'),
   port: envInt('PORT', 3001),
   adminPort: envInt('ADMIN_PORT', 3003),
+  connectorPort: envInt('CONNECTOR_PORT', 3004),
```

Comment:
> The name could be better -- the connector has several ports, and the backend will use several of them. Maybe call this one the "connector graphql port"?

I'm happy to change it, however, the backend will likely only connect to the graphql - so I'm not sure it'll ever have multiple.

> Also, should this be a host ("hostname:port") rather than just a port? Actual deployments (e.g. on kubernetes) won't necessarily share a localhost (though 127.0.0.1:3004 is a good default).

It can be, but we have no idea how this is being deployed yet. I left it out for simplicity for now as we'll likely need to fix the port situation anyway when we deploy this.

Reply:
> the backend will likely only connect to the graphql - so I'm not sure it'll ever have multiple.

The Send API will send ILP packets to the connector (over HTTP), which is a separate host/port.

> It can be, but we have no idea how this is being deployed yet.

All the more reason to support arbitrary hosts.

Reply:
@wilsonianb 

> How about referring to this and the service as IlpAccountsService?

Sorry, I missed this earlier. Basically it connects directly to the `admin-api` in the interledger service. So we thought ConnectorService was a more general fit.

Reply:
> The Send API will send ILP packets to the connector (over HTTP), which is a separate host/port.

Will that be exposed by the backend though? I thought that would be exposed directly from the connector?

> All the more reason to support arbitrary hosts.

My point was more that changing it is superfluous at this stage. But I will change it.